src/ui/ui.js

```diff
@@ -1,27 +1,29 @@
-import { cargarDataPokemones } from '../servicios/servicios.js'
-import { buscarPokemonEnLS } from '../storage/storage.js'
+import { cargarPokemon } from '../servicios/servicios.js'
+import { buscarPokemonEnLocalStorage } from '../storage/storage.js'
 import { buscarPokemonEnApi, obtenerInfoPokemones } from '../api/api.js'
+
 export async function inicializar(){
   const infoPokemones = await obtenerInfoPokemones()
   borrarPokemonesAnteriores()
-  mostrarCantidadPokemones(infoPokemones.count)
-  crearListaPokemones(infoPokemones.results)
-  console.log(infoPokemones.results)
+  mostrarCantidadPokemones(infoPokemones.totalPokemones)
+  crearListaPokemones(infoPokemones)
 }
 export function mostrarCantidadPokemones (cantidadPokemones) {
   document.querySelector('#cantidad-pokemones').textContent = cantidadPokemones
 }
-export async function crearListaPokemones (pokemones) {
+export async function crearListaPokemones(pokemones) {
   const $listaPokemones = document.querySelector('#lista-pokemones')
-    pokemones.forEach(pokemon => {
+  // PREGUNTAR SI listadoPokemones ESTA BIEN
+  const listadoPokemones = pokemones.listaPokemones
+    listadoPokemones.forEach(pokemon => {
```

Comment:
Creo que la variable está de más, no la usás en ningún lado.
```suggestion
    pokemones.listaPokemones.forEach(pokemon => {
```